packages/three-vrm-materials-mtoon/src/shaders/mtoon.frag

```diff
@@ -722,7 +790,9 @@ void main() {
       #pragma unroll_loop_start
       for ( int i = 0; i < NUM_HEMI_LIGHTS; i ++ ) {
 
-        #if THREE_VRM_THREE_REVISION >= 133
+        #if THREE_VRM_THREE_REVISION >= 157
+          irradiance += getHemisphereLightIrradiance( hemisphereLights[ i ], geometryNormal );
```

Comment:
`geometry.normal` -> `geometryNormal` , defined in `<lights_fragment_begin>`

looks good